src/infrastructures/projectFile/migration.ts

```diff
@@ -186,9 +186,16 @@ export const migrateProjectFileObject = async (
             voice.engineId === audioItem.engineId &&
             voice.styleId === audioItem.styleId,
         );
-        if (voice == undefined)
-          throw new Error(`voice == undefined: ${oldEngineId}, ${oldStyleId}`);
-        audioItem.voice = voice;
+        if (voice == undefined) {
+          // Voiceの情報が見つからなかった場合は適当なSpeakerIdを入れておく
```

Comment:
マルチエンジンをオンにした状態でエンジンを抜いてからアプデ、で製品版でも起こりうるので変えました。

Reply:
これ理由をNOTEしておかないと理由忘れてthrowするようにデグれやすい気がしました！

コメント内容はwhatで、それはコードから読み取れるので、どっちかというとwhyを書くと良さそう。
```suggestion
          // Voiceの情報が見つからなかった場合は適当なSpeakerIdを入れておく
          // NOTE: マルチエンジンの追加エンジンを削除してアップデートした際にundefinedになる
```

あるいはWHYだけでも良いかも。
```suggestion
          // NOTE: マルチエンジンの追加エンジンを削除してアップデートした際にundefinedになる
```